### PR TITLE
fix: regex char not allowed, change api names

### PR DESCRIPTION
**Pull request by zachferland**
Api names produced regex errors (dash char), changed names and generated api again

### CODE REVIEW

File: api-server/src/server/mod.rs [hunk 1]
```diff
@@ -21,27 +21,25 @@ pub use crate::context;
...
+use crate::{Api, CeramicEventsPostResponse, CeramicSubscribeSortkeySortvalueGetResponse};
 
 mod paths {
     use lazy_static::lazy_static;
 
     lazy_static! {
         pub static ref GLOBAL_REGEX_SET: regex::RegexSet = regex::RegexSet::new(vec![
             r"^/ceramic/events$",
-            r"^/ceramic/subscribe/(?P<sort-key>[^/?#]*)/(?P<sort-value>[^/?#]*)$"
+            r"^/ceramic/subscribe/(?P<sortkey>[^/?#]*)/(?P<sortvalue>[^/?#]*)$"
```
Comment (zachferland) [inline api-server/src/server/mod.rs:32]: "error: invalid capture group character" thrown before, removed dash

File: api/src/lib.rs [hunk 2]
```diff
@@ -11,8 +11,8 @@ pub async fn start(
     peer_id: PeerId,
     network: Network,
     addr: String,
-    interest: impl Recon<Key = Interest>,
-    model: impl Recon<Key = EventId>,
+    interest: impl Recon<Key = Interest> + 'static,
```
Comment (zachferland) [inline api/src/lib.rs:14]: not sure what the compiler was complaining about here

File: api-server/src/server/mod.rs (hunk 1 above)
Comment (nathanielc) [reply api-server/src/server/mod.rs:32]: Yeah I ran into that yesterday too, thanks for the fix.

**Review by nathanielc — CHANGES_REQUESTED** [review]

File: api-server/api/openapi.yaml [hunk 3]
```diff
@@ -10,21 +10,21 @@ info:
 servers:
 - url: /
 paths:
-  /ceramic/subscribe/{sort-key}/{sort-value}:
+  /ceramic/subscribe/{sortkey}/{sortvalue}:
```
[the remark this answers is not in the document]
Comment (zachferland) [reply api-server/api/openapi.yaml]: yeah see it makes auto names better, just updated in latest commit

**Review by nathanielc — APPROVED** [review]

**Review by AaronGoldman — APPROVED** [review]

**Pull request merged**